### PR TITLE
fix missing `<my_github_username>`

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ This covers how to setup and configure a development environment using the Forkl
 A Katello development environment can be deployed on CentOS 6 or 7. Ensure that you have followed the steps to setup Vagrant and the libvirt plugin. There are a variety of useful development environment options that should or can be set when creating a development box. These options are designed to configure your environment ready to use your own fork, and create pull requests. To create a development box:
 
   1. Copy boxes.yaml.example to boxes.yaml. If you already have a boxes.yaml, you can copy the entries in boxes.yaml.example to your boxes.yaml.
-  2. Now, replace <my_github_username> with your github username
+  2. Now, replace `<my_github_username>` with your github username
```
Comment (ekohl) [inline docs/development.md]: I wonder if the backticks are sufficient and we should remove the braces. @ehelms what do you think?

**Comment by ekohl** [comment]
I like this, but I think there are more things in the file that need it as well. All the --options below it for example.

File: docs/development.md (hunk 1 above)
Comment (ehelms) [inline docs/development.md]: :+1:

**Comment by tbrisker** [comment]
@ekohl thanks, added more backticks.

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**